dashboard/app/controllers/foorm_preview_controller.rb

```diff
@@ -52,7 +52,8 @@ def name
       regional_partner_name: params[:regionalPartnerName] || "Regional Partner A",
       is_virtual: params[:isVirtual] == 'true' || false,
       is_friday_institute: params[:isFridayInstitute] == 'true' || false,
```

Comment:
Ah I should have looked at this stuff when getting rid of Friday Institute option in UI, LMK if this can be removed as well.

Reply:
I think it's ok to leave for now since we have surveys that use this variable--we can either remove when summer surveys are done or keep it in there since it won't hurt anything